client/src/components/SearchedRecipeTile.js

```diff
@@ -1,11 +1,12 @@
 import React from "react"
+import { Link } from "react-router-dom"
 
 const SearchedRecipeTile = (props) => {
   return(
-    <div>
+    <Link to={`/search/${props.result.id}`}>
       <h3>{props.result.title}</h3>
       <img src={props.result.image} alt={`A picture of ${props.result.title}`}/>
-    </div>
+    </Link>
```

Comment:
Looking good! as long as the child elements of Link/a are not interactable, this nesting is great!